ios/browser/api/ads/ads_client_ios.mm

```diff
@@ -59,8 +61,8 @@
   return [bridge_ getAdEvents:ad_type confirmationType:confirmation_type];
 }
 
-void AdsClientIOS::ResetAdEvents() const {
-  [bridge_ resetAdEvents];
+void AdsClientIOS::ResetAdEventsForId(const std::string& id) const {
+  [bridge_ resetAdEventsForId:id];
```

Comment:
You missed updating the bridge delegate, there is no method `resetAdEventsForId`

Reply:
Fixed